### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by AndreyEKravchenko** [PHPOffice/PhpSpreadsheet#3549] [General]
can I install two libraries together? I need to use php 5.0 and 8.1 at the same time...

**Comment by MarkBaker** [comment]
Not easily, No! Not unless you scope one of those releases to a different namespace. But PhpSpreadsheet never supported PHP 5.0. The first release had a minimum requirement of PHP 5.6

**Reply by AndreyEKravchenko** [reply]
sorry... i make mistake.. 5.6 and 8.1
